integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -530,7 +537,18 @@ def export(self, **kwargs):
 
         # If we're not at the max payload count, we can check for orphaned content and shove it in the delete payload
         current_payload_count = len(items_to_create) + len(items_to_update) + len(items_to_delete)
+
+        self._log_info(
+            f'Exporter finished iterating over catalogs for customer: {self.enterprise_customer.uuid}.'
+            f'With a current payload count of: {current_payload_count}, checking for orphaned content.'
```

Comment:
nit: how about we nix the "checking for orphaned content", because we don't know if we're actually going to do that yet, and instead log `max_payload_count`?